src/main/default/lwc/lookup/lookup.html

```diff
@@ -146,13 +148,15 @@
             </template>
             <!-- Multi-selection end -->
 
-            <!-- Errors start -->
-            <template for:each={errors} for:item="error">
-                <label key={error.id} role="alert" class="slds-form-element__label slds-var-m-top_xx-small form-error"
-                    >{error.message}</label
-                >
+            <!-- Error start -->
+            <template if:true={hasErrors}>
```

Comment:
this if should not be necessary, looping through an empty list results in the same output.
the best would be to revert this change so we are not spreadingt to far away from the original branch